### PR TITLE
Add cargo packages for windows

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
# Description
- Add cargo-outdated
- Add cargo-audit

#### Related issue:
https://github.com/actions/virtual-environments/issues/1029

## Images:
- [windows16](https://github.visualstudio.com/virtual-environments/_build/results?buildId=74845&view=results)
- [windows19](https://github.visualstudio.com/virtual-environments/_build/results?buildId=74844&view=results)

image
image

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1 [hunk 1]
```diff
@@ -37,6 +37,21 @@ function Get-RustVersion {
...
+    return & cbindgen --version
+}
+
+function Get-CargoauditVersion {
+    return & cargo audit --version
+}
+
+function Get-CargooutdatedVersion {
+    return & cargo outdated --version
+}
```
Comment (miketimofeev) [inline images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1:54]: nitpick: new line here

File: images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1 [hunk 2]
```diff
@@ -37,6 +37,21 @@ function Get-RustVersion {
     return "Rust ${rustVersion}"
 }
 
+function Get-BindgenVersion {
+    return & bindgen --version
```
Comment (al-cheb) [inline images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1:41]: Why do we need &?

Comment (dmitry-shibanov) [reply images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1:41]: I think we don't need, just to be consistent with other checks in file. Would it be better to remove `&` ?

File: images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1 (hunk 1 above)
Comment (dmitry-shibanov) [reply images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1:54]: Added new line

File: images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1 (hunk 2 above)
Comment (dmitry-shibanov) [reply images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1]: Removed &

File: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 [hunk 3]
```diff
@@ -38,6 +38,14 @@ $markdown += New-MDList -Style Unordered -Lines @(
     (Get-NodeVersion)
 )
 
+$markdown += New-MDHeader "Rust packages:" -Level 3
```
Comment (vsafonkin) [inline images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1:41]: I believe this section will be better to place after tools section, not on top docs

File: images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1 [hunk 4]
```diff
@@ -37,6 +37,22 @@ function Get-RustVersion {
...
+function Get-CbindgenVersion {
+    return cbindgen --version
+}
+
+function Get-CargoauditVersion {
+    return cargo audit --version
+}
+
+function Get-CargooutdatedVersion {
+    return cargo outdated --version
```
Comment (MaksimZhukov) [inline images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1:53]: Perhaps `Get-CargoAuditVersion` and `Get-CargoOutdatedVersion` names will be more readable.

File: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 (hunk 3 above)
Comment (dmitry-shibanov) [reply images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1]: Moved after tools cli

File: images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1 (hunk 4 above)
Comment (dmitry-shibanov) [reply images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1]: Fixed naming

**Pull request merged**